doc/btp_aics.txt

```diff
@@ -0,0 +1,92 @@
+Audio Input Control Service (ID 10)
+==================================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read supported commands
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Auido Input Control Service Init
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+
+		This command is used to register and initialize AICS in IUT.
+
+		In case of an error, the error reponse will be returned.
+
+	Opcode 0x03 - Set gain
+		Controller Index:	<controller id>
+		Command parameters:	Gain (1 octet)
```

Comment:
"Response parameters:    <none>"     (same for other commands that return only status)

Reply:
Done